packages/features/form-builder/FormBuilder.tsx

```diff
@@ -654,6 +659,11 @@ function VariantFields({
         label={t("identifier")}
       />
 
+      <CheckboxField
```

Comment:
Added support for name field as well which is a variantsSupportedField and has a bit different UI.